org.envirocar.app/res/layout/fragment_tracklist.xml

```diff
@@ -19,7 +19,9 @@
     with the enviroCar app. If not, see http://www.gnu.org/licenses/.
 
 -->
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/coordinator_layout"
```

Comment:
Is there a good reason for changing the layout?

Reply:
yes, if we use a co-ordinator layout it takes care of snack bar, floating action bar and bottom navigation bar not collide.
before:

![before](https://user-images.githubusercontent.com/75211982/145403313-72df1c09-982a-48cb-b83a-9d0d303e0fb4.jpeg)

after:

![after](https://user-images.githubusercontent.com/75211982/145403440-d6fbc0b0-4db9-40a0-80c8-0c74a9a871f8.jpeg)